src/apps/search-result/facet-browser.test.ts

```diff
@@ -0,0 +1,247 @@
+    cy.get(`[aria-label="Close facet browser modal"]`).click();
+  });
+
+  it.only("Check if Subject has all its terms", () => {
+    cy.contains(".disclosure__text", "Subject").click();
+    cy.contains(".facet-browser__facet-group button", "vokal");
+    cy.contains(".facet-browser__facet-group button", "instrumental");
+    cy.contains(".facet-browser__facet-group button", "jazz");
+    cy.contains(".facet-browser__facet-group button", "rock");
+    cy.contains(".facet-browser__facet-group button", "Harry Potter");
+    cy.contains(".facet-browser__facet-group button", "magi");
+    cy.contains(".facet-browser__facet-group button", "troldmænd");
+    cy.contains(".facet-browser__facet-group button", "swing");
+    cy.contains(".facet-browser__facet-group button", "krimi");
+    cy.contains(".facet-browser__facet-group button", "fantasy");
+    cy.contains(".disclosure__text", "Subject").click();
+    cy.get(`[aria-label="Close facet browser modal"]`).click();
+  });
+
+  it.only("Check if Work types has all its terms", () => {
+    cy.contains(".disclosure__text", "Work types").click();
+    cy.contains(".facet-browser__facet-group button", "Bøger");
+    cy.contains(".facet-browser__facet-group button", "Musik");
+    cy.contains(".facet-browser__facet-group button", "Film");
+    cy.contains(".facet-browser__facet-group button", "Noder");
+    cy.contains(".facet-browser__facet-group button", "Anmeldelser");
+    cy.contains(".facet-browser__facet-group button", "Periodika");
+    cy.contains(".facet-browser__facet-group button", "Spil");
+    cy.contains(".disclosure__text", "Work types").click();
+    cy.get(`[aria-label="Close facet browser modal"]`).click();
+  });
+
+  it.only("Checks the logic of selected terms and open facets", () => {
+    cy.log("Check total results");
+    cy.contains("h1", "“harry” (703)");
+
+    cy.log("Click on creators, select Joanne K. Rowling");
+
+    cy.contains("Creators").click();
+    cy.interceptGraphql({
+      operationName: "searchFacet",
+      fixtureFilePath:
+        "search-result/facet-browser/searchFacet_terms_joanne-k-rowling"
+    });
+    cy.interceptGraphql({
+      operationName: "searchWithPagination",
+      fixtureFilePath:
+        "search-result/facet-browser/searchWithPagination_terms_joanne-k-rowling"
+    });
+    cy.contains("button", "Joanne K. Rowling").click();
+
+    cy.log("Close modal and check for new results");
+    cy.get(`[aria-label="Close facet browser modal"]`).click();
+    cy.contains("h1", "“harry” (36)");
+
+    cy.log(
+      "Open modal and check if creators are opened and Joanne K. Rowling is selected"
+    );
+    cy.contains("button", "+ MORE FILTERS").click();
+    cy.get(`[aria-controls="facet-creators"]`).should(
+      "have.attr",
+      "aria-expanded",
+      "true"
+    );
+    cy.contains("button", "Joanne K. Rowling").should(
+      "have.class",
+      "tag--outlined-selected"
+    );
```

Comment:
Please use aria attributes instead of CSS classes to check state.

This goes for the entire test.